docs/install.md

```diff
@@ -1302,16 +1304,16 @@ active volumes.
 A PMEM-CSI Deployment's `status` field is a `DeploymentStatus` object, which
 carries the detailed state of the driver deployment. It is comprised of [deployment
 conditions](#deployment-conditions), [driver component status](#driver-component-status),
-and a `phase` field. The phase of a Deployment is a high-level summary
-of where the Deployment is in its lifecycle.
+and a `phase` field. The phase of a PMEM-CSI deployment is a high-level summary
+of where the the PmemCSIDployment is in its lifecycle.
 
 The possible `phase` values and their meaning are as below:
 
 | Value | Meaning |
 |---|---|
 | empty string | A new deployment. |
 | Running | The operator has determined that the driver is usable<sup>1</sup>.  |
-| Failed | For some reason the state of the `Deployment` failed and cannot be progressed. The failure reason is placed in the `DeploymentStatus.Reason` field. |
+| Failed | For some reason the state of the `PmemCSIDeployment` failed and cannot be progressed. The failure reason is placed in the `DeploymentStatus.Reason` field. |
```

Comment:
Just noticed this: a state cannot fail...

"For some reason, the `PmemCSIDeployment` failed and currently cannot proceed."

Reply:
Done.